### ISSUE TITLE
Option to assign ticket directly to "Catchall" company instead of creating a new company

### ISSUE THREAD

**Issue by zpeters** [prontotools/gravityformsconnectwise#11]
I might be using the plugin wrong but for our use case what i'm trying to do is setup a generic form that an anonymous user could submit a ticket.  What i'm trying to do is gather all of the contact information and actually just add that all to the ticket body (i can do this).  In the background i'm using hidden fields to map the Company / Contact to a default we use called "Catchall" that is manually reviewed.

This works, however i always get an error that the company already exists (see below).

Is there a way to create an option to not try to make a new Company/Contact and just use a default one?  I'd be willing to help write this, but i want to make sure i'm not missing something that already exists.

Thanks

URL => https://api-na.myconnectwise.net/v4_6_release/apis/3.0/company/companies
data => {
    "id": 0,
    "identifier": "Catchall",
    "name": "Catchall",
    "addressLine1": "-",
    "addressLine2": "-",
    "city": "-",
    "state": "-",
    "zip": "-",
    "phoneNumber": null,
    "faxNumber": null,
    "website": null,
    "type": {
        "id": "1"
    },
    "status": {
        "id": "1"
    }
}
Response[code] => 400
Response[body] => {
  "code": "InvalidObject",
  "message": "company object is invalid",
  "errors": [
    {
      "code": "ObjectExists",
      "message": "Company ID already in use.",
      "resource": "company",
      "field": "identifier"
    }
  ]
}
---------
Configure error notifications: [Redacted]

Install the Gravity Forms Logging Add-on to view the complete error log: https://www.gravityhelp.com/documentation/article/logging-add-on/

**Comment by corydbrown** [comment]
@zpeters You're right, and error shouldn't be thrown when a duplicate company exists. For your case I think we might already have a solution.

In the feed settings, if you don't map Company to anything, it'll default to a company with the ID "catchall". 

![screen shot 2016-08-09 at 11 18 56 am](https://cloud.githubusercontent.com/assets/2689853/17504446/1913cb12-5e23-11e6-87ac-a23dc7ae07c9.png)

If that's the same catchall ID you're using it should work. 

Can you please test that out and let us know if that works? Thank you!

**Comment by zpeters** [comment]
I can confirm that if i leave Company mapped to "Select a Field" it does create a ticket and the ticket gets assigned to "Catchall".

I am now getting a 500 error after the submission.  It seems as if the "Thank you" page doesn't load.

Any ideas what that might be?

**Comment by corydbrown** [comment]
@zpeters Hmm no that's weird.

What does the log say after you submit the form?

[Gravity Forms Logging plugin](https://www.gravityhelp.com/documentation/article/logging-add-on/)
